src/DotLiquid/DotLiquid.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyTitle>DotLiquid</AssemblyTitle>
     <NeutralLanguage>en-US</NeutralLanguage>
     <Authors>Tim Jones;Alessandro Petrelli</Authors>
-    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net45;</TargetFrameworks>
```

Comment:
This has significant implication for consumers of the library. They no longer will be able to use dot liquid with older .NET standard. In the past this has been a blocking issue for PRs to merge to master.

Reply:
We have net45, are we supporting older than .NET 4.5?

Reply:
I've reincluded 1.3 and made a dedicated build for 2.0 to take advantage of the performance gains. Even running the test suite is about 33% longer on 1.3 vs 2.0